tower-http/src/services/fs/serve_dir.rs

```diff
@@ -376,6 +470,13 @@ fn empty_body() -> ResponseBody {
     ResponseBody::new(body)
 }
 
+fn body_from_bytes(bytes: Bytes) -> ResponseBody {
+    let body = BoxBody::new(Full::from(bytes))
+        .map_err(|err| match err {})
+        .boxed();
```

Comment:
The initial `BoxBody::new` shouldn't be necessary since there is a `.boxed()` at the end.

```suggestion
    let body = Full::from(bytes)
        .map_err(|err| match err {})
        .boxed();
```